### ISSUE TITLE
INPformer++何时开源

### ISSUE THREAD

**Issue by lzxssz** [luow23/INP-Former#39]
作者你好，你们的工作非常棒，请问扩展的INPformer++何时开源

**Comment by luow23** [comment]
嗨咯，你好，我们适用的就说draem的dataloader，不过一个值得注意的点，默认使用Draem的dataloader的num worker只能设为0，这可能会导致加载速度下降，你可以参考这个[issue](https://github.com/VitjanZ/DRAEM/issues/8)来解决这个问题。

**Comment by wuxie12345** [comment]
> 感谢您的关注。鉴于近期项目任务繁重，代码整理与开源工作暂时延后，后续有时间将尽快安排处理。

请问什么时候能开源呢？

**Comment by luow23** [comment]
十张真实异常样本和合成样本混合到一起传给模型,  在分割头的训练过程中，只使用了异常样本监督

**Comment by hyao1** [comment]
> 嗨咯，你好，我们使用的就是draem的dataloader，不过一个值得注意的点，默认使用Draem的dataloader的num worker只能设为0，这可能会导致加载速度下降，你可以参考这个[issue](https://github.com/VitjanZ/DRAEM/issues/8)来解决这个问题。

这个问题好奇怪，我在a6000服务器上不会报错，在windows10上用2080ti就会报错，你们是在linux下跑的吗

**Comment by luow23** [comment]
不是的，是在windows下跑的

**Comment by shyyxy** [comment]
嗨喽，作者你好，加入分割头的Lseg在训练中是要和Lnpm = Lsm + λLsc加权到一起嘛，代码中应该这样写嘛
`loss= loss + 0.2 * g_loss+βloss_seg` ，如果是，那β要怎么确认

**Comment by luow23** [comment]
嗨咯，你好，β设为1

**Comment by shyyxy** [comment]
非常感谢
